### PR TITLE
Changed some stuff to protected

### PR DESCRIPTION
**Pull request by J3fftw1**
## Description
Changed some stuff to protected

## Changes
Changed some stuff to protected
Added some annotations 

## Related Issues
N/A

## Checklist

- [x] I have fully tested the proposed changes and promise that they will not break everything into chaos.
- [x] I have also tested the proposed changes in combination with various popular addons and can confirm my changes do not break them.
- [x] I followed the existing code standards and didn't mess up the formatting.
- [ ] I did my best to add documentation to any public classes or methods I added.
- [x] I have added `Nonnull` and `Nullable` annotations to my methods to indicate their behaviour for null values
- [ ] I added sufficient Unit Tests to cover my code.

### CODE REVIEW

File: src/main/java/io/github/thebusybiscuit/slimefun4/implementation/SlimefunPlugin.java [hunk 1]
```diff
@@ -198,15 +198,17 @@ else if (getServer().getPluginManager().isPluginEnabled("CS-CoreLib")) {
 
             // Set up localization
             getLogger().log(Level.INFO, "Loading language files...");
-            local = new LocalizationService(this, config.getString("options.chat-prefix"), config.getString("options.language"));
+            local = new LocalizationService(this, config.getString("options.chat-prefix"),
+                config.getString("options.language"));
```
Comment (LinoxGH) [inline src/main/java/io/github/thebusybiscuit/slimefun4/implementation/SlimefunPlugin.java:202]: Please break this after `this` not in the middle of a method.

File: src/main/java/io/github/thebusybiscuit/slimefun4/implementation/items/androids/ButcherAndroid.java [hunk 2]
```diff
@@ -34,7 +34,8 @@ protected void attack(Block b, BlockFace face, Predicate<LivingEntity> predicate
         double damage = getTier() >= 3 ? 20D : 4D * getTier();
         double radius = 4.0 + getTier();
 
-        for (Entity n : b.getWorld().getNearbyEntities(b.getLocation(), radius, radius, radius, n -> n instanceof LivingEntity && !(n instanceof ArmorStand) && !(n instanceof Player) && n.isValid() && predicate.test((LivingEntity) n))) {
+        for (Entity n : b.getWorld().getNearbyEntities(b.getLocation(), radius, radius, radius, n -> n
+            instanceof LivingEntity && !(n instanceof ArmorStand) && !(n instanceof Player) && n.isValid() && predicate.test((LivingEntity) n))) {
```
Comment (LinoxGH) [inline src/main/java/io/github/thebusybiscuit/slimefun4/implementation/items/androids/ButcherAndroid.java:38]: Please break this in a more reasonable location. Like after all `radius` args.

**Review by LinoxGH — CHANGES_REQUESTED** [review]

**Review by LinoxGH — APPROVED** [review]
Better.

**Comment by LinoxGH** [comment]
Build is failing.

**Comment by J3fftw1** [comment]
because of the merge of master will check why

**Comment by J3fftw1** [comment]
something fucked up this branch locally for me i will make a new pr

**Pull request closed without merging**